qualtran/simulation/classical_sim.py

```diff
@@ -265,3 +265,20 @@ def format_classical_truth_table(
         for invals, outvals in truth_table
     ]
     return '\n'.join([heading] + entries)
+
+
+def signed_addition(a: int, b: int, N: int, is_signed: bool) -> int:
```

Comment:
Args section to document the arguments; I'd suggest renaming `N` to `mod`, and I'd suggest changing `mod` and `is_signed` to be keyword-only arguments (by putting `..., *, mod, is_signed)`. Baring that, I would strongly recommend _using_ keyword args when calling this function to make the flags obvious

Reply:
done ... though I prefer to pass `num_bits` rather than `N` or `mod`